module/VuFind/src/VuFind/Db/Table/AccessToken.php

```diff
@@ -126,6 +128,18 @@ public function getNonce(int $userId): ?string
         return null;
     }
 
+    /**
+     * Save access token.
+     *
+     * @param AccessTokenEntityInterface $accessToken Access token
+     *
+     * @return void
+     */
+    public function save(AccessTokenRow $accessToken): void
```

Comment:
The [current Doctrine work in #2233](https://github.com/vufind-org/vufind/pull/2233/files#diff-9fa4f8d7b9ca434b2067e542970768c2312c3fa6e1b177a92f28b99a2ffa5433R88) has a generic persistEntity method in the AbstractDbService class. Would it make sense to implement the same pattern here? Obviously it's not exactly right to call `save()` on an instance of `EntityInterface`, but we could do a type check on the incoming parameter to see if it's a Laminas Row object and call save if it is -- and throw an exception if we receive something unexpected.